### PR TITLE
Update and inithook

### PR DESCRIPTION
**Pull request by a3s7p**
Closes https://github.com/turnkeylinux/tracker/issues/554

### CODE REVIEW

File: overlay/usr/lib/inithooks/bin/plone.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/python
+"""Set Magento admin password and email, and domain to serve
```
Comment (JedMeister) [inline overlay/usr/lib/inithooks/bin/plone.py]: Needs minor adjustment :smile:

**Comment by JedMeister** [comment]
As noted, the python inithook needs a minor adjustment. It's not really that big a deal (as it's not displayed to the user) but may cause confusion.

**Comment by a3s7p** [comment]
@JedMeister, :arrow_up:

**Comment by JedMeister** [comment]
:+1:

**Pull request merged**